content/en/docs/contribute/advanced.md

```diff
@@ -19,7 +19,7 @@ client and other tools for some of these tasks.
 
 ## Be the PR Wrangler for a week
 
-SIG Docs [approvers](/docs/contribute/participating/#approvers) take regular turns as the PR wrangler for the repository and are added to the [PR Wrangler rotation scheduler](https://github.com/kubernetes/website/wiki/PR-Wranglers#2019-schedule-q1q2) for weekly rotations. 
+SIG Docs [approvers](/docs/contribute/participating/#approvers) take week-long turns [wrangling PRs](https://github.com/kubernetes/website/wiki/PR-Wranglers#2019-schedule-q1q2) for the repository.
```

Comment:
Lines 11-12, Fix links!

Reply:
As a general note: After addressing this round of comments, I'm going to go back through the entire section and sweep for links.

That said, the link to approvers works, and the [wiki](https://github.com/kubernetes/website/wiki/PR-Wranglers) still only shows wranglers for Q1 and Q2. I've removed the `#2019-schedule-q1q2` from this link in the hopes of future-proofing it, however.